Source/com/IUnknown.h

```diff
@@ -148,7 +148,7 @@ namespace ProxyStub {
             , _parent(parent)
             , _channel(channel)
             , _remoteReferences(1)
-            , _name(&(name[7])) /* It always starts with "struct ", remove it...*/
```

Comment:
This, I think, came for the windows platform (all had "struct " in front of it and I had to drop it. Suggest to "call" the new method from above _string Demangle(const TCHAR name[])_  where for windows, it is demangled but you stripe of the struct keyword, and for Linux you really demangle? Than the Proxy will give the right name and you do not need to adpat the controller. This way everyone will profit from a proper name of the interface ;-)